lib/index.js

```diff
@@ -126,11 +126,10 @@ function* createEverything() {
 
   var adminTestDatabaseClient = pgp(adminTestDatabase);
   yield adminTestDatabaseClient.task(alterPrivileges);
-
-  pgp.end();
```

Comment:
This is there intentionally (I didn't really know what I was doing so asked in that direction without a lot of context [here](https://github.com/vitaly-t/pg-promise/issues/284)). I just enabled Travis CI on this repo, although without any assertions or anything so the status is meaningless, but it reproduces the issue nicely:

```
$ web-app-db
Creating admin user...
Creating application user...
Creating test database
Dropping the public schema and replacing it with a new one...
Enabling the case-insensitive text extension (citext)...
Setting the default schema for admin user
Setting the default schema for application user
Granting application user usage on schema...
Revoking functions privileges from public user...
Granting tables permissions to application user...
Granting sequences permissions to application user...
Granting tables functions to application user...
Cloning test database to create development database...

error: source database "min_auth_test" is being accessed by other users
```

I think you should be able to see the build [here](https://travis-ci.org/tobymurray/web-app-db/builds/222136789)?